src/global/Screens.js

```diff
@@ -74,13 +88,13 @@ const Screens = {
     screen: 'Pangea.NationsScreen',
     label: i18n.t('screens.nations.tabTitle'),
     icon: AssetsImages.TabIcons.nations,
-    title: i18n.t('screens.nations.title'),
+    title: '',
```

Comment:
Is this correct? Just asking since it's an title which is an empty string.

Reply:
Yes, that screens don't have a title (it's shown as large title in a custom way).